spec/integration/explicit_encryption_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe 'Explicit Encryption' do
+  require_libmongocrypt
+
+  let(:client) { ClientRegistry.instance.new_local_client(['localhost:27017']) }
+  let(:key_vault_namespace) { 'test.keys' }
+
+  let(:client_encryption_opts) do
+    {
+      kms_providers: {
+        local: { key: Base64.encode64("ru\xfe\x00" * 24) }
+      },
+      key_vault_namespace: key_vault_namespace
+    }
+  end
+
+  it 'encrypts and decrypts a value' do
+    client_encryption = Mongo::ClientEncryption.new(
+      client,
+      client_encryption_opts
+    )
+
+    data_key_id = client_encryption.create_data_key
```

Comment:
create_data_key documents return value thusly:

<pre>
    # @return [ BSON::Binary ] UUID representing the data key _id
</pre>

Is the data key id something a user is expected to store somewhere, such as in an environment variable? If so, how does the user convert the data key id to and from a simple string?

Reply:
That's a good point. I'm changing the API to accept a data key as a string.